pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/project.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "Microsoft.NETCore.Portable.Compatibility": "1.0.2",
-    "Microsoft.NETCore.Runtime.CoreCLR": "1.1.0",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.1.1-servicing-24720-01",
```

Comment:
The only place where UWP package can actually ship from, right now, is release/1.0.0 branch since it requires components from closed build. 

FWIW, UWP was not shipped from release/1.1.0.